crates/driver/src/print_build.rs

```diff
@@ -542,14 +548,9 @@ async fn print_file(
 
                 apply_binaries(&mut t, metadata, module_config, &element)?;
 
-                let _handle = concurrent_io_operations.acquire().await?;
                 let sub_target = opt.working_directory.join(node).join("BUILD.bazel");
                 emitted_files.push(sub_target.clone());
-                tokio::fs::write(&sub_target, t.emit_build_file()?)
-                    .await
-                    .with_context(|| {
-                        format!("Attempting to write file data to {:?}", target_file)
-                    })?;
+                on_child(sub_target, t);
```

Comment:
Does this do anything? `on_child` returns a future that we discard, so it would be a no-op i think?

Reply:
Good catch. on_child is the callback, so it should be `on_child(sub_target, t)?;`?